PubSub/src/BatchPublisher.php

```diff
@@ -64,11 +64,28 @@ class BatchPublisher
      */
     private $client;
 
+    /**
+     * @var bool
+     */
+    private $enableCompression;
+
+    /**
+     * @var int
+     */
+    private $compressionBytesThreshold;
+
     /**
      * @param string $topicName The topic name.
-     * @param array $options [optional] Please see
-     *        {@see Google\Cloud\PubSub\Topic::batchPublisher()} for
+     * @param array $options [optional] {
+     *        Please @see Google\Cloud\PubSub\Topic::batchPublisher() for
```

Comment:
Should it be like this? I am not sure how these would render.
```suggestion
     *        Please {@see Google\Cloud\PubSub\Topic::batchPublisher()} for
```

Reply:
yes, that's correct. But this path won't render - it needs to either be the fully quantified path or the relative path. So either 
```suggestion
     *        Please {@see PubSub\Topic::batchPublisher()} for
```

or 
```suggestion
     *        Please {@see \Google\Cloud\PubSub\Topic::batchPublisher()} for
```